README.md

```diff
@@ -1 +1,22 @@
-# angular-writing-a-custom-directive-lab
\ No newline at end of file
+# Writing a custom Directive
+
+## Objectives
+
+- Create a tabs Directive
+- Create a tab Directive
+- Inherit Directive Controller methods into child Directives
+- Make tabs clickable
+- Show clicked tabs content when clicked
+- Ensure an initial tab's content can be displayed on initialisation
+
+## Instructions
+
+Let's create some awesome tabs.
```

Comment:
Any good examples of tab navigation that we could show here? That would help set a fun context for this lab.